Core/Tracking.lua

```diff
@@ -102,8 +110,7 @@ end
 -- TODO: What's up with the Hyacinth Macaw? Leaving it at "None" might be a better default?
```

Comment:
This can definitely be removed if you also retire the good old Macaw :) ("None" isn't to be taken literally)